unit_testing/test_sct_compute_mtsat.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+from __future__ import print_function, absolute_import
+
+import os
+import sys
+
+from spinalcordtoolbox.utils import sct_test_path
+from spinalcordtoolbox import __sct_dir__
+sys.path.append(os.path.join(__sct_dir__, 'scripts'))
```

Comment:
Is this needed? If these tests are meant to be run by `pytest` only I don't think this is necessary.

Reply:
indeed, i thought it was needed for the `import sct_compute_mtsat` below, but i removed these lines and it worked fine.  Fixed in 312a6bdecefd88a0901715450f2da23672a08ad4